viewing_party/utils_03.py

```diff
@@ -0,0 +1,24 @@
+def get_unique_watched(user_data):
+    watched = set([movie['title'] for movie in user_data['watched']])
+
+    friends_watched = set([movie['title']
+        for friend in user_data['friends']
+            for movie in friend['watched']
+    ])
```

Comment:
Notice that this code is used again on lines 16-19. Consider writing a helper function to encapsulate this functionality to DRY (don't repeat yourself) up your code.